autorest/codegen/models/code_model.py

```diff
@@ -412,3 +412,14 @@ def enums_filename(self) -> str:
         if self.options["version_tolerant"] or self.options["low_level_client"]:
             return "_enums"
         return f"_{self.module_name}_enums"
+
+    @property
+    def is_legacy(self) -> bool:
+        return not (
+            self.options["version_tolerant"] or self.options["low_level_client"]
+        )
+        # return not any(
```

Comment:
some commented out code here

Reply:
I know there's a lot of places where we do this check, it would be great if you could update those to use this property instead. Great idea to add this to the code model!

Reply:
Yes, I will do that :)